docs/IP/ip.rst

```diff
@@ -0,0 +1,10 @@
+
+************************
+Intellectual Properties
```

Comment:
I ended up going with `IPs / Macros` -- might be better as `IP and Macros`?

Reply:
Your second suggestion looks way more better. I will change to it.